spp_custom_field/models/cus_partner.py

```diff
@@ -20,14 +20,17 @@ def _get_view(self, view_id=None, view_type="form", **options):
         if view_type == "form":
             doc = arch
             basic_info_page = doc.xpath("//page[@name='basic_info']")
-
+            group_view = self.env["ir.ui.view"].search([("id", "=", view_id)])
+            group_view_name = self.env.ref("g2p_registry_group.view_groups_form").name
+            is_group = False
+            if group_view.name == group_view_name:
+                is_group = True
```

Comment:
@emjay0921 I think it's better to use the context in the ir.actions.act_window to get if a form is a group or individual since it is already setup in the xml. In this, you don't need to depend on the module `g2p_registry_group`

```suggestion
            action_id = self.env["ir.actions.act_window"].browse(options.get("action_id"))
            if basic_info_page:
                is_group = action_id.context.get("default_is_group", False)
```

Reply:
I tried your approach and it's good though it doesn't work with the `action_id.context.get()` as it was a string so I converted it first to dict.